app/dashboard/page.tsx

```diff
@@ -1,6 +1,6 @@
 "use client";
 
-import { SimpleForm } from "@/components/SimpleForm";
+import { SimpleForm } from "@/components/BasicInfo";
```

Comment:
question: Does this even work? Surprised that it doesn't fail because there's no named export called `SimpleForm` from that file